### DISCUSSION TITLE
Error:   cannot import name 'Stream' ?

### DISCUSSION THREAD

**Discussion by MaximAbdulatif** [tweepy/tweepy#1408] [Questions] [answered]
Very annoying error (google didn't help): 
![Screenshot 2020-07-23 at 13 33 31](https://user-images.githubusercontent.com/20270945/88284020-ed12c580-ccec-11ea-8afe-a8e065c37f84.jpg)

**Comment by Harmon758** [comment, marked as answer]
First of all, you'll want to regenerate your credentials now.

If this file you're editing is named tweepy.py and in your system path, it's recursively importing itself rather than the Tweepy library.
Otherwise, another file in your system path is named tweepy.py and recursively importing itself after it's imported on the first line.
You'll want to change your filename(s) or remove them from your system path so that they're not imported prior to Tweepy itself.

**Comment by MaximAbdulatif** [comment]
Ooh, how naïve of me 🙃 
Thank you so much indeed @Harmon758
